SlevomatCodingStandard/Sniffs/Classes/PropertyPerClassLimitSniff.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\ClassHelper;
+use SlevomatCodingStandard\Helpers\SniffSettingsHelper;
+use function sprintf;
+use const T_CLASS;
+use const T_TRAIT;
+
+final class PropertyPerClassLimitSniff implements Sniff
+{
+
+	public const CODE_PROPERTY_PER_CLASS_LIMIT = 'PropertyPerClassLimit';
+
+	/** @var int */
+	public $maxCount = 10;
```

Comment:
I suggest to rename this property to `$maxPropertyCount`, because it is more consistent with the naming of other count/length/limit properties in this repository.